lib/aws_recon/collectors/organizations.rb

```diff
@@ -13,13 +13,18 @@ def collect
     #
     # describe_organization
     #
-    @client.describe_organization.each do |response|
-      log(response.context.operation_name)
+    begin
+      @client.describe_organization.each do |response|
+        log(response.context.operation_name)
 
-      struct = OpenStruct.new(response.organization.to_h)
-      struct.type = 'organization'
+        struct = OpenStruct.new(response.organization.to_h)
+        struct.type = 'organization'
 
-      resources.push(struct.to_h)
+        resources.push(struct.to_h)
+      end
+    rescue Aws::Organizations::Errors::AWSOrganizationsNotInUseException => e
+      log_error(e.code)
+      return resources
```

Comment:
Optionally suppress or raise, as in [L57](https://github.com/darkbitio/aws-recon/blob/729eebfb47f4ca0a92f72069f2e8d181c8a28ed1/lib/aws_recon/collectors/organizations.rb#L57)